src/test/java/org/opentripplanner/routing/fares/OrcaFareServiceTest.java

```diff
@@ -0,0 +1,199 @@
+            PIERCE_COUNTY_TRANSIT,
+            SOUND_TRANSIT,
+            COMM_TRANS_COMMUTER_EXPRESS
+        );
+        List<Ride> trip4 = getTrip(
+            EVERETT_TRANSIT,
+            COMM_TRANS_COMMUTER_EXPRESS,
+            SOUND_TRANSIT
+        );
+        List<Ride> trip5 = getTrip(
+            KC_WATER_TAXI_VASHON_ISLAND,
+            KC_METRO
+        );
+        List<Ride> trip6 = getTrip(
+            SEATTLE_STREET_CAR,
+            KC_METRO,
+            COMM_TRANS_COMMUTER_EXPRESS
+        );
+        return Stream.of(
+            Arguments.of(trip1, Fare.FareType.regular, DEFAULT_RIDE_PRICE_IN_CENTS),
+            Arguments.of(trip1, Fare.FareType.senior, DEFAULT_RIDE_PRICE_IN_CENTS),
+            Arguments.of(trip1, Fare.FareType.youth, DEFAULT_RIDE_PRICE_IN_CENTS),
+            Arguments.of(trip1, Fare.FareType.orcaLift, 200f),
+            Arguments.of(trip1, Fare.FareType.orcaRegular, DEFAULT_RIDE_PRICE_IN_CENTS),
+            Arguments.of(trip1, Fare.FareType.orcaSenior, 200f),
+            Arguments.of(trip1, Fare.FareType.orcaYouth, 300f),
+            // The following costs are made up of the expected cost to be applied for each agency. This is usually the
+            // most expensive leg of the journey.
+            Arguments.of(trip2, Fare.FareType.orcaLift, 0f + DEFAULT_RIDE_PRICE_IN_CENTS + 125f),
+            Arguments.of(trip2, Fare.FareType.orcaRegular, 0f + DEFAULT_RIDE_PRICE_IN_CENTS + DEFAULT_RIDE_PRICE_IN_CENTS),
+            Arguments.of(trip2, Fare.FareType.orcaSenior, 0f + DEFAULT_RIDE_PRICE_IN_CENTS + 125f),
+            Arguments.of(trip2, Fare.FareType.orcaYouth, 200f + DEFAULT_RIDE_PRICE_IN_CENTS + 0f),
+            Arguments.of(trip3, Fare.FareType.orcaLift, DEFAULT_RIDE_PRICE_IN_CENTS + 0f + 0f),
+            Arguments.of(trip3, Fare.FareType.orcaRegular, 0f + DEFAULT_RIDE_PRICE_IN_CENTS + 0f),
+            Arguments.of(trip3, Fare.FareType.orcaSenior, 0f + 0f + 200f),
+            Arguments.of(trip3, Fare.FareType.orcaYouth, 0f + 0f + 300f),
+            Arguments.of(trip4, Fare.FareType.orcaLift, 0f + 200f + 0f),
+            Arguments.of(trip4, Fare.FareType.orcaRegular, 0f + DEFAULT_RIDE_PRICE_IN_CENTS + 0f),
+            Arguments.of(trip4, Fare.FareType.orcaSenior, 0f + 0f + 200f),
+            Arguments.of(trip4, Fare.FareType.orcaYouth, 0f + 0f + 300f),
+            Arguments.of(trip5, Fare.FareType.orcaLift, 450f + 0f),
+            Arguments.of(trip5, Fare.FareType.orcaRegular, 575f + 0f),
+            Arguments.of(trip5, Fare.FareType.orcaSenior, 300f + 0f),
+            Arguments.of(trip5, Fare.FareType.orcaYouth, 450f + 0f),
+            Arguments.of(trip6, Fare.FareType.orcaLift, 0f + 0f + 200f),
+            Arguments.of(trip6, Fare.FareType.orcaRegular, 0f + 0f + DEFAULT_RIDE_PRICE_IN_CENTS),
+            Arguments.of(trip6, Fare.FareType.orcaSenior, 0f + 0f + 200f),
+            Arguments.of(trip6, Fare.FareType.orcaYouth, 0f + 0f + 300f)
```

Comment:
These trips don't have tests for `regular`, `youth` and `senior` FareTypes. Should they?

Reply:
I've added them back in. I left them out because they will alway use the default ride price.